JASP-Tests/R/tests/testthat/test-ancova.R

```diff
@@ -344,13 +344,13 @@ test_that("Field - Chapter 6 results match", {
   
   
   # descriptive plots
-  plot1 <-  results$state$figures[[1]]$obj
+  plot1 <-  results$state$figures[[1]]$obj$subplots$mainPlot
```

Comment:
you should be able to pass `results$state$figures[[1]]$obj` directly to `expect_equal_plots`. Internally, that will recognize it as a JASPgraphsplot and test each subplot independently.

Reply:
I tried this but it does not work so Im keeping it like this

Reply:
Did you update both JASPgraphs and jasptools? Because at lines 98-105 of [test-ttestbayesianindependentsamples.R](https://github.com/jasp-stats/jasp-desktop/blob/development/JASP-Tests/R/tests/testthat/test-ttestbayesianindependentsamples.R) it works fine for the t-tests.